test/printResult-process.js

```diff
@@ -2,6 +2,14 @@
 
 const autocannon = require('../autocannon')
 const exampleResult = require('./fixtures/example-result.json')
+const crossArgv = require('cross-argv')
 
-const resultStr = autocannon.printResult(exampleResult)
+let opts = null
+
+if (process.argv.length > 2) {
+  const args = crossArgv(process.argv.slice(2))
+  opts = autocannon.parseArguments(args)
+}
+
+const resultStr = autocannon.printResult(exampleResult, opts)

```

Comment:
I'm not sure this change to the tests is actually being used anywhere. Can you please confirm?

Reply:
Looks like I forgot to add a file. Fixed. Thanks.